src/main/java/uk/gov/hmcts/cmc/claimstore/config/CmcDBConfiguration.java

```diff
@@ -46,6 +46,7 @@ public TransactionAwareDataSourceProxy transactionAwareDataSourceProxy(
     private void migrateFlyway(DataSource dataSource) {
         Flyway.configure()
             .dataSource(dataSource)
+            .baselineOnMigrate(true)
```

Comment:
Is there anything in place - code or practice - to mitigate the risk described in the documentation?

> Be careful when enabling this as it removes the safety net that ensures Flyway does not migrate the wrong database in case of a configuration mistake!

Reply:
I think this might for the multiple database. We have currently only single database CMC.

Reply:
I don't think we need to enable this here.